mods/species/ascent/datum/antagonist.dm

```diff
@@ -35,13 +36,13 @@
 		H.name = H.real_name
 
 /decl/special_role/hunter/equip(var/mob/living/carbon/human/player)
+	if(player?.species.get_root_species_name(player) == SPECIES_MANTID_GYNE)
+		rig_type = /obj/item/rig/mantid/gyne
+	else
+		rig_type = initial(rig_type)
 	. = ..()
 	if(.)
-		if(player.species.get_root_species_name(player) == SPECIES_MANTID_GYNE)
-			equip_rig(/obj/item/rig/mantid/gyne, player)
-		else
-			equip_rig(/obj/item/rig/mantid, player)
-			player.put_in_hands(new /obj/item/gun/energy/particle)
+		player.put_in_hands(new /obj/item/gun/energy/particle)
```

Comment:
This is technically not 1:1 but hunter is unplayable and I have plans to port the redone mode soon so w/e.